src/util/event-emitter.js

```diff
@@ -21,14 +19,14 @@ export class EventEmitter{
   }
 
   next(value){
-    this._subject.onNext(value);
+    this._subject.next(value);
   }
 
   throw(error){
-    this._subject.onError(error);
+    this._subject.error(error);
   }
 
   return(){
-    this._subject.onCompleted();
+    this._subject.complete();
   }
 }
```

Comment:
The motivation for the change to `event-emitter.js` is just to keep parity with ng2 or is it fixing something?